vendor/engines/secure_rooms/app/presenters/secure_rooms/link_collection_extension.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  module LinkCollectionExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      insert_index = tab_methods.index(:admin_products) || -1
+      tab_methods.insert(insert_index, :admin_occupancies)
+    end
+
+    def admin_occupancies
+      if single_facility? && SecureRoom.exists?
```

Comment:
We should restrict this to there being a secure room in the current facility. We don't need to bother all the other cores with a "Rooms" tab they'll never use.